src/taipy/core/config/config.schema.json

```diff
@@ -417,6 +406,14 @@
               "type": "string",
               "taipy_function": true
             }
+          },
+          "sequences": {
+            "description": "The dictionary of sequence descriptions.",
+            "type": "object",
+            "items": {
+              "type": "string"
+            },
+            "uniqueItems": true
```

Comment:
Should we add a version in the TAIPY block ?

Reply:
I dont have any comments on that :/ @jrobinAV what do you think?

Reply:
I don't know yet, because we will need to maintain this. Can we discuss it in a dedicated discussion in a separate ticket/PR?